### PR TITLE
[blocked] Test run tally

### PR DESCRIPTION
**Pull request by tmpolaczyk**
This PR tries to add tests for all the `RadError`s that do not have a conversion to `RadonError`.

### CODE REVIEW

**Comment by tmpolaczyk** [comment]
This PR introduces a `getResolvedDrs` subcommand in the node CLI. Not sure if we want that to be used by end users, but where do we add dev-specific subcommands?

**Comment by aesedepece** [comment]
This is on hold until the undergoing refactor of `run_tally_report`.

File: src/cli/node/json_rpc_client.rs [hunk 1]
```diff
@@ -15,7 +15,8 @@ use log::*;
 use std::collections::HashMap;
 use std::convert::{TryFrom, TryInto};
 use witnet_data_structures::chain::{
-    DataRequestOutput, Environment, OutputPointer, PublicKeyHash, Reputation, ValueTransferOutput,
+    Block, DataRequestInfo, DataRequestOutput, Environment, OutputPointer, PublicKeyHash,
+    Reputation, ValueTransferOutput,
 };
 use witnet_data_structures::proto::ProtobufConvert;
```
Comment (lrubiorod) [inline src/cli/node/json_rpc_client.rs]: fmt

File: src/cli/node/json_rpc_client.rs [hunk 2]
```diff
@@ -15,7 +15,8 @@ use log::*;
 use std::collections::HashMap;
 use std::convert::{TryFrom, TryInto};
```
Comment (lrubiorod) [inline src/cli/node/json_rpc_client.rs]: fmt

**Comment by lrubiorod** [comment]
would be great to ask for a specific data request resolved

**Comment by aesedepece** [comment]
What's the blocker for this?

**Comment by aesedepece** [comment]
> What's the blocker for this?

Ping

**Comment by aesedepece** [comment]
> > What's the blocker for this?
> 
> Ping

Ping, ping!

Third ping is a PR close.

**Comment by aesedepece** [comment]
> > > What's the blocker for this?
> > 
> > 
> > Ping
> 
> Ping, ping!
> 
> Third ping is a PR close.

Ping, ping, ping!

You can't say I didn't warn you.

**Pull request closed without merging**